common/core/src/endpoint.ts

```diff
@@ -47,7 +47,7 @@ export function moduleEventPath(deviceId: string, moduleId: string): string {
 }
 
 export function moduleMessagePath(deviceId: string, moduleId: string): string {
-  return modulePath(deviceId, moduleId) + '/messages/devicebound';
```

Comment:
>devicebound [](start = 53, length = 11)

you'll need to update _endpoint_test.js

Reply:
Doh!  How did I miss that?  I thought I ran the tests too!

---
In reply to: [214413537](https://github.com/Azure/azure-iot-sdk-node/pull/354#discussion_r214413537) [](ancestors = 214413537)